### PR TITLE
Add more groupings

### LINKED ISSUE #26: New groupings
Pedantic mode groupings (which I can do when #20 is merged!)

- Group cognac, grappa, pisco, under brandy
- Add Curacao and group it under triple sec (maybe add cointreau and grand marnier too, IBA differentiation them all)
- Ungroup simple syrup from sugar (in unpedantic mode it now disappears from syrups and goes into seasoning: sugar). Instead, make simple syrup the grouping for simple syrup and gomme syrup
- Group ginger beer and ginger ale

---

Other random thinking about ingredients

- I vanilla extract bitters? It includes lots of alcohol
- I think instead of peach puree it should list peaches, and should be made fresh.

### PR DESCRIPTION
**Pull request by wooorm**
Closes GH-26.

These are all things people have strong opinions on, and they aren’t very important imo. they make sense to me though, but let me know if I should change something.

### CODE REVIEW

File: front/src/Data.elm [hunk 1]
```diff
@@ -1074,7 +1090,7 @@ recipes =
       , ingredients =
             [ { material = gin, quantity = CL 4.5 }
             , { material = lemonJuice, quantity = CL 3 }
-            , { material = gommeSyrup, quantity = CL 1 }
+            , { material = simpleSyrup, quantity = CL 1 }
```
Comment (tmcw) [inline front/src/Data.elm]: Okay so _technically_ gomme syrup has gum arabic in it, but from what I'm hearing even bartenders use simple syrup in its place, so this swap works for me.

File: front/src/Data.elm [hunk 2]
```diff
@@ -1385,7 +1401,7 @@ recipes =
             , { material = cognac, quantity = CL 1 }
             , { material = angosturaBitters, quantity = Dash 2 }
             , { material = sugar, quantity = Cube 1 }
-            , { material = tripleSec, quantity = FewDrops } -- Optional
+            , { material = grandMarnier, quantity = FewDrops } -- Optional
```
Comment (tmcw) [inline front/src/Data.elm]: 👍 after looking into it, these un-triple-sec'ing decisions look legit - this recipe still refers to triple sec though, so should probably refer to Grand Marnier instead.

**Review by tmcw — CHANGES_REQUESTED** [review]
Looks good! Few minor tweaks, and then if you run the refactoring before merging master the merge conflicts shouldn't be too bad - the (vim) substitution I've used is:

```
%s/{ material = \(\w\+\), quantity = \([^}]\+\)}/ingredient \1 (\2)/g
```

File: front/src/Data.elm (hunk 1 above)
Comment (wooorm) [reply front/src/Data.elm]: yeah indeed, there is a tiny difference from what I know too, and I *believe* IBA used to mention gomme years ago, but searching today yields no results, so it seems they solved the problem by using simple syrup everywhere!

**Comment by tmcw** [comment]
❇️!

**Review by tmcw — APPROVED** [review]

**Pull request merged**